File: tests/fixtures/boundary_ir/repos/csharp/Widget.cs
namespace App;

public interface IRenderer
{
    int Render();
}

public class Widget : IRenderer
{
    private int count;

    public Widget(int count)
    {
        this.count = count;
    }

    public int Render()
    {
        return Helper(count);
    }

    private int Helper(int value)
    {
        return value + 1;
    }
}

public enum Color
{
    Red,
    Green,
    Blue
}

File: tests/fixtures/boundary_ir/repos/java/Widget.java
package app;

public class Widget {
    private int count;

    public Widget(int count) {
        this.count = count;
    }

    public int render() {
        return helper(count);
    }

    private int helper(int value) {
        return value + 1;
    }
}

enum Color {
    RED,
    GREEN,
    BLUE
}
